app/models/ext_management_system.rb

```diff
@@ -196,7 +196,10 @@ def hostname_uniqueness_valid?
   end
 
   def hostname_format_valid?
+    return unless hostname_required?
```

Comment:
This one is already handled `validate :hostname_uniqueness_valid?, :hostname_format_valid?, :if => :hostname_required?`

Reply:
Yeah I saw that, but wasn't sure why hostname_uniqueness_valid? had the guard as well.  Figured I'd keep it consistent.

Reply:
Oh interesting, okay works for me